.github/workflows/ci-rust.yml

```diff
@@ -91,15 +91,15 @@ jobs:
           RUST_LOG: debug
         timeout-minutes: 10
 
-      - name: Check non bindings crates using sodium crypto features
+      - name: Check platform crates using sodium crypto features
```

Comment:
Agnostic is also tested here (hence we test everything except the bindings)

Reply:
Done, I missed this one